config.example.toml

```diff
@@ -1,3 +1,5 @@
+include = "../repositories"
```

Comment:
Relative path of the entry file would be better. Can you help me moving `config.example.toml` and `repositories` into config folder?

Reply:
ok